src/controllers/shop/checkout-item.controller.js

```diff
@@ -10,14 +10,23 @@ const checkoutItem = async (req, res) => {
       attributes: { exclude: ["password"] },
     });
 
-    const item = req.body;
+    const items = req.body;
 
     const transactions = [];
 
-    //for (const item of items) {
-      const { product_id, qty } = item;
+    for (const item of items) {
+      const { cart_id, qty } = item;
 
-      const product = await Product.findOne({ where: { id: product_id } });
+      const cart = await Cart.findOne({
+        where: { id: cart_id },
+      });
+      console.log(cart.id);
+      console.log(typeof cart.id);
```

Comment:
I've removed this

Reply:
refer to commit: https://github.com/Belega-Village-UNUD/backend-api-edb/pull/51/commits/63b0a8f98a2e368a94cc9eef20db18b9d6fc6ac6